lib/PuppeteerSharp.Tests/RequestInterceptionTests/RequestRespondTests.cs

```diff
@@ -145,5 +145,36 @@ public async Task ShouldStringifyInterceptedRequestResponseHeaders()
             Assert.Equal("True", response.Headers["foo"]);
             Assert.Equal("Yo, page!", await Page.EvaluateExpressionAsync<string>("document.body.textContent"));
         }
+
+        [PuppeteerTest("requestinterception.spec.ts", "Request.respond", "should allow for multiple headers to be sent for an intercepted request response ")]
```

Comment:
I don't see this test [here](https://github.com/puppeteer/puppeteer/blob/main/test/requestinterception.spec.ts). Am I missing something?

Reply:
Thanks for the review,  I was meaning to follow up on this. 

I ran out of time to dig into in the purpose of PuppeteerTestAttribute, so i just did a copy paste with the intention of following up. Should I simply remove the attribute? 

Perhaps PuppeteerTestAttribute xml doc summary could be expanded a little?

Reply:
We use `PuppeteerTest` to report how many missing tests we have compared with the upstream puppeteer.

Reply:
Thanks for the clarification 👍  Attribute has been removed.